cli/debug/debug.go

```diff
@@ -62,7 +61,7 @@ func NewCommand() *cobra.Command {
 	debugCommand.Flags().StringVarP(&fqbn, "fqbn", "b", "", tr("Fully Qualified Board Name, e.g.: arduino:avr:uno"))
 	port.AddToCommand(debugCommand)
 	debugCommand.Flags().StringVarP(&programmer, "programmer", "P", "", tr("Programmer to use for debugging"))
-	debugCommand.Flags().StringVar(&interpreter, "interpreter", "console", fmt.Sprintf(tr("Debug interpreter e.g.: %s, %s, %s, %s, %s"), "console", "mi", "mi1", "mi2", "mi3"))
+	debugCommand.Flags().StringVar(&interpreter, "interpreter", "console", tr("Debug interpreter e.g.: %s", "console, mi, mi1, mi2, mi3"))
```

Comment:
The reason I intentionally chose not to use this capability when I did the i18n campaign is because I think it makes the code unclear regarding which components will be internationalized. I could accept it if the function was somehow using the extra arguments to enhance the translation effort, but it's not. My feeling is that the `i18n.Tr` function should have one job: internationalization of strings, just as `fmt.Sprintf` has its one job.

But I do understand that my approach resulted in quite a bit of extra code that is not really necessary, even more so due to the technique of substituting out parts of the strings that should not be translated. It wasn't clear exactly how much the impact of my decision would be when I made it at the start of the work. So I think it's a reasonable change to make. I just wanted to share my decision process.

Reply:
Yep, I figured it went like that :-)